crates/config/src/inline/conf_parser.rs

```diff
@@ -2,6 +2,13 @@ use super::{remove_whitespaces, InlineConfigParserError};
 use crate::{inline::INLINE_CONFIG_PREFIX, InlineConfigError, NatSpec};
 use regex::Regex;
 
+/// Used to detect invalid configuration keys in the inline config.
+///
+/// # Example
+///
+/// forge-config: default.fuzz.runs = 100 - `fuzz` is a valid key
+/// forge-config: default.wrong.runs = 100 - `wrong` is an invalid key
+const VALID_CONFIG_KEYS: [&str; 2] = ["fuzz", "invariant"];
```

Comment:
For now this is sufficient I think, assuming this functionality will be updated when we have full support for in-line configuration